app/src/main/java/com/example/drawingactivity/CustomColorViewModel.kt

```diff
@@ -29,10 +31,20 @@ class CustomColorViewModel(
         updateViews()
     }
 
+    private var viewModelDrawnBitmap: Bitmap? = null
+
+    fun saveUserBitmap(b: Bitmap?) {
```

Comment:
Kdoc

Reply:
Kdoc added to explain what the lambda is used for